.changes/next-release/bugfix-68695d67-47ab-4fad-9ec2-0a732e88adda.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Prior to this change, while deleting a resource, the name of the resource had to be entered to delete the resource. This change requires the user to enter/type \"delete me\" instead of the resource name to delete the resource."
```

Comment:
This should be shortened greatly, also it wasn't a bug

Reply:
Changed the type to "feature" change and shortened the description.

Reply:
forgot to delete the file?